src/Catty/ViewController/Continue&New/MaintainObject/MaintainScript/BrickManager/BrickManager.h

```diff
@@ -39,7 +39,7 @@
 - (CGSize)sizeForBrick:(id<BrickProtocol>)brick;
 - (NSInteger)checkEndLoopBrickTypeForDrawing:(BrickCell*)cell;
 - (NSArray*)animateWithIndexPath:(NSIndexPath*)path Script:(Script*)script andBrick:(Brick*)brick;
-- (NSArray*)scriptCollectionCopyBrickWithIndexPath:(NSIndexPath*)indexPath andBrick:(Brick*)brick;
+- (NSArray*)scriptCollectionCopyBrickWithIndexPath:(NSIndexPath*)indexPath Brick:(Brick*)brick nestedIndexOrZero:(int)nestedInteger;
```

Comment:
As far as I can see the new `nestedIndexOrZero` parameter is only being used from within the `BrickManager`. Suggestion: make method private, leave old method signature and internally forward to .`..nestedIndexOrZero: 0` without changing the interface. You can also rename it to `netstedIndex` only, more importantly please rename the parameter `Brick` either to `brick` (lowercase) or combine all parameters with an `and`.